modules/ROOT/pages/bonita-applications-deep-dive.adoc

```diff
@@ -0,0 +1,65 @@
+= Applications deep dive 
+:description: This page explains how Bonita runs applications and presents the Bonita applications natively embedded in Bonita Runtime
+
+{description}
+
+
+[.card-section]
+== How it works in Bonita Runtime
+
+[.card.card-index]
+--
+xref:bonita-applications-interface-overview.adoc[[.card-title]#Overview# [.card-body.card-content-overflow]#pass:q[How applications are run by Bonita]#]
+--
+
+[.card.card-index]
+--
+xref:cache-configuration-and-policy.adoc[[.card-title]#Cache management# [.card-body.card-content-overflow]#pass:q[Cache policies and how to configure the caching mechanism]#]
+--
+
+[.card.card-index]
+--
+xref:log-in-and-log-out.adoc[[.card-title]#Login mechanism# [.card-body.card-content-overflow]#pass:q[How to get users to log into an application]#]
+--
+
+[.card-section]
+== Bonita applications
+Bonitasoft developers develop Bonita using Bonita. +
+This is why Bonita comes with four embedded applications, with different status: 
+
+* Core applications. They are critical to the good operation of Bonita in a Production environment. They can be customized but they cannot be deleted:
+  ** Application Directory 
+  ** Super Administrator
+* Generic applications. They are fully functional, can be customized, but can be deleted if not useful in a Production environment:
+  ** Administrator
+  ** User
+  
+[.card.card-index]
+--
+xref:application-directory-application.adoc[[.card-title]#Application directory# [.card-body.card-content-overflow]#pass:q[The logged user's catalog of applications]#]
+--
+
+[.card.card-index]
+--
+xref:super-administrator-application.adoc[[.card-title]#Super Administrator# [.card-body.card-content-overflow]#pass:q[The tool to bootstrap a Bonita Runtime, perform maintenance actions, and update the BDM]#]
+--
+
+[.card.card-index]
+--
+xref:administrator-application.adoc[[.card-title]#Administrator# [.card-body.card-content-overflow]#pass:q[A tool to manage an automation project in production (BDM excepted)]#]
+--
+
+[.card.card-index]
+--
+xref:user-application.adoc[[.card-title]#User# [.card-body.card-content-overflow]#pass:q[A tool for actual people making processes alive, centered around a generic tasklist]#]
```

Comment:
"making processes alive" I think about Dave when I reade this.
maybe
```suggestion
xref:user-application.adoc[[.card-title]#User# [.card-body.card-content-overflow]#pass:q[A tool for actual people using the processes, centered around a generic tasklist]#]
```
?

Reply:
I will go for "executing the processes". What they use is the applications, to me.

Reply:
:+1:

Reply:
"Start process instances and execute tasks using a generic tasklist"

Reply:
done